conf/bafu.conf.part

```diff
@@ -17,6 +17,91 @@ source src_ch_bafu_hydrologie-untersuchungsgebiete_stationen : def_searchable_fe
         from hydrologie.hydrologie_untersuchungsgebiete_stationen
 }
 
+source src_ch_bafu_vec25-gewaessernetz_2000 : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+        , name as label \
+        , 'feature' as origin \
+        , remove_accents(bgdi_id::text||' '||gwlnr||' '||gewissnr::text||' '||name||' '||objectval) as detail \
+        , 'ch.bafu.vec25-gewaessernetz_2000' as layer \
+        , quadindex(the_geom) as geom_quadindex \
+        , st_y(st_transform(st_centroid(the_geom),4326)) as lat \
+        , st_x(st_transform(st_centroid(the_geom),4326)) as lon \
+        , box2d(the_geom) as geom_st_box2d \
+        , bgdi_id::text as feature_id \
+    from vec25.gewaessernetz_2000
+}
+
+source src_ch_bafu_vec25-seen : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+        , name as label \
+        , 'feature' as origin \
+        , remove_accents(bgdi_id::text||' '||gewaesserkennzahl::text||' '||seetyp::text||' '||name||' '||ausgleichsbecken::text||' '||reguliert::text||' '||seeflaeche_km2::text||' '||inhalt_see_mio_m3::text||' '||nutzinhalt_mio_m3::text||' '||tiefe_see_m::text||' '||hoehenlage_muem::text||' ' ||uferlaenge_m::text||' '||gwlnr) as detail \
+        , 'ch.bafu.vec25-seen' as layer \
+        , quadindex(the_geom) as geom_quadindex \
+        , st_y(st_transform(st_centroid(the_geom),4326)) as lat \
+        , st_x(st_transform(st_centroid(the_geom),4326)) as lon \
+        , box2d(the_geom) as geom_st_box2d \
+        , bgdi_id::text as feature_id \
+    from vec25.seen
+}
+
+source src_ch_bafu_hydrologischer-atlas_flussgebiete : def_searchable_features
+{
+    sql_db = bafu_dev
+    sql_query = \
+        SELECT bgdi_id::bigint as id \
+        , name as label \
+        , 'feature' as origin \
+        , remove_accents(bgdi_id::text||' '||nummer::text||' '||name||' '||shape_area::text||' '||umfang::text) as detail \
```

Comment:
the same here, do we really want bgdi_id and shape_area and umfang as searchable attributes?
The attributes which are concatenated should be wrapped inside coalesce function:
```sql
remove_accents(bgdi_id::text||' '||nummer::text||' '||name||' '||shape_area::text||' '||umfang::text)
```
should be
```sql
remove_accents(coalesce(bgdi_id::text,' ')||' '||coalesce(nummer::text,' ')||' '||coalesce(name,' ')||' '||coalesce(shape_area::text,' ')||' '||coalesce(umfang::text,' ')) as detail
```